app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -107,80 +61,132 @@ function handleOptimisticButtRender(butt) {
   } else {
     // Handles actual following of tags/users
     try {
-      //lets try grab the event buttons info data attribute user id
+      // lets try grab the event buttons info data attribute user id
       var evFabUserId = JSON.parse(butt.dataset.info).id;
       var requestVerb = butt.dataset.verb;
-      //now for all follow action buttons
-      document.querySelectorAll('.follow-action-button').forEach(function(fab) {
+      // now for all follow action buttons
+      document.querySelectorAll('.follow-action-button').forEach(fab => {
         try {
-          //lets check they have info data attributes
+          // lets check they have info data attributes
           if (fab.dataset.info) {
-            //and attempt to parse those, to grab that buttons info user id
+            // and attempt to parse those, to grab that buttons info user id
             var fabUserId = JSON.parse(fab.dataset.info).id;
-            //now does that user id match our event buttons user id?
+            // now does that user id match our event buttons user id?
             if (fabUserId && fabUserId === evFabUserId) {
-              //yes - time to assign the same state!
+              // yes - time to assign the same state!
               assignState(fab, requestVerb);
             }
           }
         } catch (err) {
-          return;
+          return undefined;
         }
+        return undefined;
```

Comment:
eslint threw useless return, which it seemed to be. Had to add undefined to allow it or will have to remove the `try` statement too since the `catch` would be empty which led to its removal and thus the try being without any catches.